spec/lib/pdf_filler/md502_pdf_spec.rb

```diff
@@ -250,5 +250,57 @@
         expect(pdf_fields["Enter 11"].to_i).to eq intake.direct_file_data.fed_taxable_ssb
       end
     end
+
+    context "deduction" do
+      context "method" do
+        it "checks box if standard deduction" do
+          allow_any_instance_of(Efile::Md::Md502Calculator).to receive(:calculate_deduction_method).and_return "S"
+          expect(pdf_fields["Check Box 34"]).to eq "Yes"
+        end
+
+        it "does not check box if non-standard deduction" do
```

Comment:
[dustiest dust] I think this is technically called "non-taxable" deduction method 🤓

Reply:
oops i think i did start to write "not standard" in other places for this reason